### PR TITLE
unpin python 3.11

### PR DESCRIPTION
**Pull request by Mic92**
Just use whatever nixpkgs has by default

### CODE REVIEW

**Comment by Mic92** [comment]
@mergify queue

File: default.nix [hunk 1]
```diff
@@ -2,15 +2,15 @@
   pkgs ? import <nixpkgs> { },
 }:
 
-pkgs.python311.pkgs.buildPythonApplication {
+pkgs.python3.pkgs.buildPythonApplication {
   pname = "nix-update";
-  version = "1.0.0";
+  version = "1.5.2";
   src = ./.;
   format = "pyproject";
```
Comment (SuperSandro2000) [inline default.nix:9]:
```suggestion
  pyproject = true;
```

**Comment by Mic92** [comment]
@mergify queue

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**